dateparser/languages/loader.py

```diff
@@ -34,10 +36,14 @@ def _load_data(self):
             data = get_data('data', 'languages.yaml')
         else:
             data = self.file.read()
-        data = load_yaml(data)
-        base_data = data.pop('base', {})
+        
+        if not self._raw_data:
```

Comment:
*New* `LanguageDataLoader` is created for a good reason only. For now, while we not yet expose settings for the user, do you expect this to happen more that once? Also, we don't want loader to be able to load *only one data*. What if new instance was defined with different `file` ?